config/config.yaml

```diff
@@ -81,6 +81,17 @@ report:
     # select a sample sheet column for stratification
     by-column: condition
 
+# optional: standardized mutation overview plots with maf:
+# https://bioconductor.org/packages/release/bioc/vignettes/maftools/inst/doc/maftools.html
+maf:
+  activate: false
+  # sample alias from scenario.yaml for tumor sample;
+  # only necessary, if activate: true
+  tumor_alias: "tumor"
+  # optional: sample alias from scenario.yaml for normal sample
+  # normal_alias: "normal"
```

Comment:
Does this always work, also for non cancer samples?

Reply:
If so, maybe call the fields more generic, e.g. `primary_alias` and `control_alias`. Let us mayhbe have a quick chat.

Reply:
```suggestion
# optional: standardized mutation overview plots with maf:
# https://bioconductor.org/packages/release/bioc/vignettes/maftools/inst/doc/maftools.html
maf:
  activate: false
  # sample alias from scenario.yaml for tumor sample;
  # only necessary, if activate: true
  primary_alias: "tumor"
  # optional: sample alias from scenario.yaml for normal sample
  # control_alias: "normal"
```

Reply:
Good idea. The full refactoring is here:
https://github.com/snakemake-workflows/dna-seq-varlociraptor/pull/297/commits/4b1688fcd576c45da7e86b6797c210bfb245ddfb